src/Entity/Alias.php

```diff
@@ -46,6 +46,10 @@ class Alias implements SoftDeletableInterface, Stringable
     #[ORM\Column(nullable: true)]
     protected ?string $destination = null;
 
+    #[ORM\Column(length: 255, nullable: true)]
+    #[Assert\Length(max: 255)]
```

Comment:
For whatever reason, we used to do validation directly in the entity. But that's not the typical approach. Form models are designed for that. While this is already incorrect in the existing code, it should be adjusted.

Reply:
And I would reduce the validation to 30-50 characters, otherwise we'll have to make adjustments to the design to accommodate a possible 255 characters.